tests/wsl/firstrun.pm

```diff
@@ -119,7 +119,9 @@ sub run {
     assert_screen [qw(yast2-wsl-firstboot-welcome wsl-installing-prompt)], 480;
 
     if (match_has_tag 'yast2-wsl-firstboot-welcome') {
-        assert_and_click 'window-max';
+        assert_screen(['window-max', 'window-minimize']);
+        assert_and_click 'window-max' if match_has_tag 'window-max';
+        assert_and_click 'window-minimize' if match_has_tag 'window-minimize';
```

Comment:
why do we need to minimize the window?

Reply:
We don't, I should add a comment here...

The new process of installing, appears in an already maximized window, but sometimes it loses focus. So I created another needle to check if the window is already maximized and click somewhere else to bring it to focus.

Reply:
Also you can maximize window by: `send_key 'super-up';` and minimize window by `send_key 'super-down';`.

Reply:
This another ticket in progress: https://progress.opensuse.org/issues/138248